.github/workflows/main.yml

```diff
@@ -0,0 +1,80 @@
+name: main
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - ".github/workflows/docs.yml"
+      - "docs/**"
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - ".github/workflows/docs.yml"
+      - "docs/**"
+
+env:
+  SUBLIME_TEXT_VERSION: "4"
+  SUBLIME_TEXT_ARCH: x64
+  PACKAGE: lsp_utils
+  UNITTESTING_TAG: master
+
+jobs:
+  macOS:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - run: echo "$HOME/Library/Python/3.7/bin" >> $GITHUB_PATH
+      - run: curl --silent https://raw.githubusercontent.com/SublimeText/UnitTesting/master/sbin/ci.sh --output ci.sh
+      - run: |
+          sh ci.sh bootstrap
+          sh ci.sh install_package LSP main https://github.com/sublimelsp/LSP.git
+          sh ci.sh install_package LSP-pyright master https://github.com/sublimelsp/LSP-pyright.git
+          sh ci.sh install_package_control
+          subl --stay &
+          sleep 8
+          sh ci.sh run_tests
+        name: Run UnitTesting tests
+
+  Linux:
+    runs-on: ubuntu-latest
+    container: sublimetext/unittesting
+    steps:
+      - uses: actions/checkout@v1
+      - run: sudo apt update
+      - run: sudo apt install --no-install-recommends -y x11-xserver-utils python3-pip
+      - run: sh -e /etc/init.d/xvfb start
+      - run: curl -OL https://raw.githubusercontent.com/SublimeText/UnitTesting/master/sbin/github.sh
+      - run: echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - run: |
+          PATH="$HOME/.local/bin:$PATH"
+          sh github.sh bootstrap
+          rm -rf "$HOME/.config/sublime-text/Packages/lsp_utils"
+          sh github.sh install_package LSP main https://github.com/sublimelsp/LSP.git
+          sh github.sh install_package LSP-pyright master https://github.com/sublimelsp/LSP-pyright.git
+          sh github.sh install_package_control
+          rm -rf "$HOME/.config/sublime-text/Packages/lsp_utils"
+          mkdir -p "$HOME/.config/sublime-text/Packages/lsp_utils"
+          cp -r ./ "$HOME/.config/sublime-text/Packages/lsp_utils"
```

Comment:
What is achieved by removing the lsp_utils folder,
than re-creating it again with the same files?

Reply:
Trying stuff out to workaround a blocking issue. See https://github.com/SublimeText/UnitTesting/issues/193